### DISCUSSION TITLE
Place an image as logo inside the form

### DISCUSSION THREAD

**Discussion by nes74-bh** [opengisch/QField#5644] [Announcements]
Hi, as the title says I want to place an image inside the form and that it appears when I take the data in my cell phone. I have tried html widget but it does not work.

The image is in local path, in a folder inside my “project” folder that says “logo” and my image is called “imagen-logo.png”.

I would be grateful if you could give me suggestions on what I can do or if it is possible to do it? Thanks in advance.

**Comment by VxTedxV** [comment]
Did you set your project in Qgis with relative paths rather than absolute paths ?

**Reply by nes74-bh** [reply]
yes

**Comment by nirvn** [comment]
@nes74-bh , I'd suggest going for a QML widget here, it integrates much better with feature form. HTML widgets will have weird overlay issues as soon as you scroll your form.

**Reply by nes74-bh** [reply]
Thanks
